src/reducers/bookmarkReducer/index.js

```diff
@@ -0,0 +1,50 @@
+import {
+  BOOKMARK_SUCCESS, UNBOOKMARK_SUCCESS, BOOKMARK_LIST_FETCHED, BOOKMARK_FAIL, 
+} from "../../actions/types";
+
+const initialState = {
+  bookmarks: [],
+  isBookmarked: false,
+};
+
+const getIsBookmarked = (bookmarks, slug) => {
```

Comment:
Great work @Peace-Apple, I love the way you use this filter function, Looks clean and readable

Reply:
Thanks